src/commands/code-analyzer/run.ts

```diff
@@ -1,19 +1,74 @@
-import {SfCommand} from '@salesforce/sf-plugins-core';
+import {Flags, SfCommand} from '@salesforce/sf-plugins-core';
+import {RunAction} from '../../lib/actions/RunAction';
+import {BundleName, getMessage} from '../../lib/messages';
 
-export default class Run extends SfCommand<void> {
+export default class RunCommand extends SfCommand<void> {
 	// We don't need the `--json` output for this command.
 	public static readonly enableJsonFlag = false;
-	public static readonly summary = 'summary boop'; // TODO
-	public static readonly description = 'description boop'; // TODO
-	public static readonly examples = [ // TODO
-		'example1 boop',
-		'example2 boop'
+	public static readonly summary = getMessage(BundleName.RunCommand, 'command.summary');
+	public static readonly description = getMessage(BundleName.RunCommand, 'command.description');
+	public static readonly examples = [
+		getMessage(BundleName.RunCommand, 'command.examples')
 	];
 
-	public static readonly flags = {};
+	public static readonly flags = {
+		// === Flags pertaining to targeting ===
+		workspace: Flags.string({
+			summary: getMessage(BundleName.RunCommand, 'flags.workspace.summary'),
+			char: 'w',
+			multiple: true,
+			delimiter: ',',
+			default: ['.']
+		}),
+		'path-start': Flags.string({
```

Comment:
String instead of file, because we still want method-level targeting (unless I'm misremembering).

Reply:
Correct. It can either be a folder, a file, or a file#method syntax. So string is needed here... and we'll validate afterward.